docs/man/rpmbuild.1.scd

```diff
@@ -42,14 +42,13 @@ code and recipe necessary to produce binary packages.
 *--recompile*
 	Compatibility aliases for *-ra*.
 
-Packages are built in four phases: parse, build, assembly and cleanup. The
-middle two are further divided into _stages_, listed below, which can be built
-separately. Building a _STAGE_ means executing all the preceding stages up to
-(and including) the one specified, unless stated otherwise.
+_STAGE_ is a single-letter argument that specifies the target stage to build.
+All preceding stages are also executed unless stated otherwise. See *BUILD
+PROCESS* for a complete overview.
 
-## Assembly stages
-The assembly stages produce packages and are the primary way of interacting with
-*rpmbuild*. If in doubt, choose one of these.
+## Creating packages
+
+Use these _STAGE_ arguments to produce final packages ready for distribution.
```

Comment:
This doesn't hold for the `r` stage though. Maybe `r` should just be moved below this?

Reply:
Hmm, never mind, just a confused thought :sweat_smile: 

Basically just meant that 'r' should maybe be in a group of its own similar to the "Use these...". Or just drop the "final, ready for distribution" classification because that's what conflicts with 'r'. These steps all produce packages, their finality and so on is a whole other question.